ozaria/site/components/interactive/PageInteractive/BaseInteractiveTitle.vue

```diff
@@ -0,0 +1,40 @@
+<script>
```

Comment:
Should we move BaseDraggableSlot and BaseInteractiveTitle to PageInteractive/common folder?

Reply:
Great idea!